chroma-manager/Makefile

```diff
@@ -208,6 +208,13 @@ rpms: cleandist tarball
 	rm -rf _topdir
 	echo "jenkins_fold:end:Make Manager RPMS"
 
+install_requirements: requirements
+	echo "jenkins_fold:start:Install Python requirements"
+	pip install --upgrade pip;                              \
+	pip install --upgrade setuptools;                       \
+	pip install -r requirements.txt
+	echo "jenkins_fold:end:Install Python requirements"
+
```

Comment:
My patch applies to the top-level `Makefile`.  You want the similar technique in the `chroma-manager/Makefile`, so even when mine lands, it will not apply the technique for `chroma-manager/Makefile`.

So you still need that in this patch, but I'm just suggesting to use this technique to avoid wasting more time on each build/test run as the time to do builds and tests has been expressed as taking too long many times already.  That is partly because we waste time running through processes which end up being a NOOP which we can avoid.  This would be another one of those.

You can see the technique as I have applied it in the top-level `Makefile` [here](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/34/commits/626489658c35e5125007e6bda3d56e3ea0692e74#diff-b67911656ef5d18c4ae36cb6741b7965R125)